plugin/ochttp/server.go

```diff
@@ -70,6 +70,12 @@ type Handler struct {
 	// from the information found in the incoming HTTP Request. By default the
 	// name equals the URL Path.
 	FormatSpanName func(*http.Request) string
+
+	// IsHealthEndpoint holds the function to use for determining if the
+	// incoming HTTP request should be considered a health check. If set and
+	// returns true, no trace will be recorded, otherwise the private
+	// isHealthEndpoint function will be used.
```

Comment:
The IsHealthEndpoint is an 'Additional' check on top of default isHealthEndpoint check. I think the doc should reflect that. Currently it seems that it will use User supplied over the default.

Reply:
This is somewhat ambiguous since the current order of the conditional could be confusing if users expect the func to have complete control over skipping tracing. If instead we flipped the order around to the following it would act as an override instead of an additional check:

`h.IsHealthEndpoint == nil && isHealthEndpoint(r.URL.Path) || h.IsHealthEndpoint(r.URL.Path)`

But then users would have to remember to include `/healthz` and `/_ah/health` when supplying a func for `IsHealthEndpoint`.

Which is preferred: the 'Additional' check currently implemented in the PR or giving the user complete control when they supply a func?

Reply:
I agree that the override/additional behavior is somewhat unclear so the comment has been to indicate that either one of the user-supplied func or the private default could cancel tracing.